### ISSUE TITLE
tldr error

### ISSUE THREAD

**Issue by 31ishita12** [higgood/seechat-folks#16]
**Context**
I am using it for the 10th time. It has worked perfectly before, but today it seems to give an error that says "tldr"

**Describe the bug**
'tldr'
Traceback:
File "/users/visitor/moustafa/.local/lib/python3.11/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 541, in _run_script
    exec(code, module.__dict__)
File "/net/fs-v2.cs.lan/export/other/visitor/moustafa/idea-factory-demo/src/demo.py", line 551, in <module>
    app()
File "/net/fs-v2.cs.lan/export/other/visitor/moustafa/idea-factory-demo/src/demo.py", line 515, in app
    context = generate_context(
              ^^^^^^^^^^^^^^^^^
File "/net/fs-v2.cs.lan/export/other/visitor/moustafa/idea-factory-demo/src/demo.py", line 260, in generate_context
    return answer_question_chatgpt(
           ^^^^^^^^^^^^^^^^^^^^^^^^
File "/net/fs-v2.cs.lan/export/other/visitor/moustafa/idea-factory-demo/src/utils/utils.py", line 141, in answer_question_chatgpt
    context = create_context_chatgpt(question, df, k=k)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/net/fs-v2.cs.lan/export/other/visitor/moustafa/idea-factory-demo/src/utils/utils.py", line 175, in create_context_chatgpt
    + str(row["tldr"])
          ~~~^^^^^^^^
File "/local/pkg/python/root-python-3.7/envs/idea-factory/lib/python3.11/site-packages/pandas/core/series.py", line 1007, in __getitem__
    return self._get_value(key)
           ^^^^^^^^^^^^^^^^^^^^
File "/local/pkg/python/root-python-3.7/envs/idea-factory/lib/python3.11/site-packages/pandas/core/series.py", line 1116, in _get_value
    loc = self.index.get_loc(label)
          ^^^^^^^^^^^^^^^^^^^^^^^^^
File "/local/pkg/python/root-python-3.7/envs/idea-factory/lib/python3.11/site-packages/pandas/core/indexes/base.py", line 3655, in get_loc
    raise KeyError(key) from err

**[Required] Where did you encounter the bug**

[ 1] While Idea Factory was thinking or generating text


**[Required] Reality vs. Expectation**
I do not understand this section but I was waiting to read about a review on trace level detection of nucleic acid but all i got was these errors.

**Screenshots**
<img width="761" alt="Screenshot 2024-04-01 at 8 59 49 AM" src="https://github.com/higgood/seechat-folks/assets/88451331/aa6eb992-adc4-405f-8325-7de6fd32b690">.

**Desktop (if applicable):**
 - OS: [macOS Sonoma]




Thank you for improving the Idea Factory!!

**Comment by vivek-verma202** [comment]
I am facing the same issue, is the site down, or has the URL been changed?

**Comment by wammar** [comment]
@31ishita12 @vivek-verma202 

Sorry for the inconvenience and the lack of follow up on my part. I have news:

The BAD: We are no longer supporting https://demo-idea-factory.ngrok.dev/  

The GOOD: We were able to resolve this issue a while back (so I'm marking this as closed).

The AWESOME: The MVP proved useful for a lot of the people who gave it a spin so we decided to implement it from scratch with a better design and a scalable architecture. The new and improved product will be launched soon at seechat.ai but we're dark-launching it at https://demo.seechat.ai and of course we'd love our early adopters to keep us honest and tell us how to improve it. The usability is not great right now, but we promise we're working hard on it and your input (positive or negative will help us prioritize what to work on.)
